src/Pelagos/Bundle/AppBundle/Resources/views/Search/results.html.twig

```diff
@@ -1,36 +1,71 @@
+    <div class="col-md-offset-4">
+        <h2 data-content="{{ count }}" id="count">
+            {{ count }} results found for: <span class="text-navy">"{{ query }}"</span>
+        </h2>
+    </div>
+
     <div class="row">
-        <div class="col-lg-12">
-            <div class="ibox float-e-margins">
-                <div class="ibox-content">
-                    <h2 data-content="{{ count }}" id="count">
-                        {{ count }} results found for: <span class="text-navy">"{{ query }}"</span>
-                    </h2>
-
-                    <div class="search-form">
-                        {% include 'PelagosAppBundle:Search:form.html.twig' %}
-                    </div>
-                    {% for result in results %}
-                        {% include 'PelagosAppBundle:Search:dataset-row.html.twig'  %}
-                    {% endfor %}
-
-                    <div class="text-center">
-                        <div class="btn-group">
-                            <div class="wrapper">
-                                <div class="container">
-                                    <div class="row">
-                                        <div class="col-sm-12">
-                                            <nav aria-label="Search results pages">
-                                                <ul id="search-pagination" class="pagination"></ul>
-                                            </nav>
-                                        </div>
+        <div class="search-form">
+            {% include 'PelagosAppBundle:Search:form.html.twig' %}
+        </div>
+    </div>
+
+
+    <div class="row">
+        <div class="col-md-2 col-xs-2">
+
+            <div class="aggregations">
+                <h5 style="margin-bottom: 5px;"><strong style="color: #337ab7;">Research Groups</strong></h5>
+
+                {% for group in researchGroupsInfo %}
+
+                <ul class="browse-list list-unstyled long-list" style="margin-bottom: 0;">
+                    <li for="{{ group.id }}">
+                        <div class="checkbox block" style="margin-top: 0; margin-bottom: 0;">
+                            <label>
+                                <input class="rg-checkbox" type="checkbox" id="{{ group.id }}">
+                                    <span class="groupName">{{ group.name }} </span> ({{ group.count }})
+                                    <div class="tooltiptext" style="display:none">
+                                        <p>{{ group.name }}</p>
                                     </div>
-                                </div>
+                            </label>
+                        </div>
+                    </li>
+                </ul>
+                {% endfor %}
+            </div>
+        </div>
+
+        <div class="col-md-10 col-xs-10">
+            <div class="breadcrumbs"></div>
+            <div class="clearfix"></div>
+            <table class="table">
```

Comment:
I would just put these table elements inside the template, for readability.

Reply:
also you could ass class: `table-striped`?

Reply:
I think the results file should be refactored, with the proper bootstrap grid layouts